patternfly/patternfly-elements
# fix: set font-family for triggers and tray headers

## set font-family styles for triggers and tray headers

* Link(s) to demo pages where this element can be viewed:

---

### What has changed and why
Bootstrap.css provides font-family styles for h1-h6 elements that are overriding slotted content styles such as navigation triggers and navigation tray titles. This pull request applies font-family styles that are more specific than the bootstrap selectors.

### Testing instructions

1. Go to this page like this one that contains the pfe-navigation web comoponent and has the bootstrap.css: https://web-dev-drupal-webux2.dev.a1.vary.redhat.com/en/about/company/management/arun-oberoi
2. Check the navigation item triggers at desktop breakpoints (these should be h3s). I expect to see that this `font-family: "RedHatText", "Overpass", Overpass, Helvetica, Arial, sans-serif;`
3. Now open the Products tab and look at the tray headers in the inspect (these should be h4s).  I expect to see that this `font-family: "RedHatText", "Overpass", Overpass, Helvetica, Arial, sans-serif;`
4. If I navigate from either of these pages to the home page I should not see a discernible difference in the font


#### Browser requirements
Your component should work in all of the following environments:


Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
